### PR TITLE
Added get_description method to XMLState

### PR DESCRIPTION
**Pull request by schaefi**
In preparation to access the contents of the <description>
section this pull requests adds a simple method to read
the contents.

### CODE REVIEW

**Comment by schaefi** [comment]
Also adds some style refactoring in the XMLState tests (breaking up too long lines)

File: test/unit/xml_state_test.py [hunk 1]
```diff
@@ -156,8 +163,12 @@ def test_set_repository(self):
         assert self.state.xml_data.get_repository()[0].get_type() == 'type'
         assert self.state.xml_data.get_repository()[0].get_alias() == 'alias'
         assert self.state.xml_data.get_repository()[0].get_priority() == 1
-        assert self.state.xml_data.get_repository()[0].get_imageinclude() is True
-        assert self.state.xml_data.get_repository()[0].get_package_gpgcheck() is False
+        assert self.state.xml_data.get_repository()[
+            0
```
Comment (dcermak) [inline test/unit/xml_state_test.py]: [Minor nitpick] I really don't like the formatting here, I think it obscures that the first element of `get_repository()` is taken. (This applies to all the other lines too).

Comment (schaefi) [reply test/unit/xml_state_test.py]: :smile: ok, you are right... just a sec

Comment (schaefi) [reply test/unit/xml_state_test.py]: how about it now ?

Comment (dcermak) [reply test/unit/xml_state_test.py]: Much better, thanks!

**Pull request merged**